wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SelectCommand.java

```diff
@@ -52,8 +55,7 @@ public SelectCommand(Map<Object, Command> commands, Supplier<Object> selector) {
   @Override
   public void initialize() {
     if (!m_commands.containsKey(m_selector.get())) {
-      m_selectedCommand =
-          new PrintCommand("SelectCommand selector value does not correspond to any command!");
+      m_selectedCommand = m_defaultCommand;
     } else {
       m_selectedCommand = m_commands.get(m_selector.get());
```

Comment:
Query the map with `getOrDefault` instead of the `if` statement, and then the selector is only invoked once.